### ISSUE TITLE
Issue with background color

### ISSUE THREAD

**Issue by MeyerBender** [sagar87/spatialproteomics#49]
The background color works when only a single channel is selected, but is behaving weirdly with two or more channels. Example:

```
plt.figure(figsize=(10, 10))
_ = img_full.pp[['DAPI', 'Na/K ATPase']].pl.colorize(['orange', 'deepskyblue'], background="darkgray").pl.autocrop().pl.show()
plt.xticks([])
plt.yticks([])
plt.tight_layout()
```

![image](https://github.com/sagar87/spatialproteomics/assets/62450528/faea26ba-6ac0-49d2-91e2-69db62e0e458)

**Comment by MeyerBender** [comment]
Need to check if this only affects `pl.colorize()` or if `pl.render_segmentation()` needs fixing as well.

**Comment by MeyerBender** [comment]
Fixed in #58.
